### PR TITLE
Add Distributed Tracing to Node docs

### PR DESCRIPTION
**Pull request by onurtemizkan**
Potentially Resolves: #5647 (Do we need to open another PR on [getsentry/develop](https://github.com/getsentry/develop)?)

Adds basic examples about:

- How to obtain a trace header from a span using `toTraceparent`
- How to use `extractTraceparentData` to continue upstream service traces.

Also removed the part that mentions `continueFromHeaders` function (which is not available in `sentry-javascript`) for JavaScript and Node docs. 

---

As an aside note, Node docs in general seem to need some restructuring, and I just remembered there is unfinished work in https://github.com/getsentry/sentry-docs/issues/3605.

We previously made a few updates in https://github.com/getsentry/sentry-docs/pull/3693, which have already crossed out some sub-tasks there.

I'm up to finish that next, if the team agrees.

### CODE REVIEW

File: src/platform-includes/performance/connect-services/node.mdx [hunk 1]
```diff
@@ -0,0 +1,27 @@
+## Distributed Tracing
+
+To obtain a trace from a span, use `toTraceParent()` method. This method returns a string that can be used as a value of the `traceparent` HTTP header.
```
Comment (imatwawana) [inline src/platform-includes/performance/connect-services/node.mdx:3]:
```suggestion
To obtain a trace from a span, use `toTraceParent()` method. This method returns a string that can be used as a value of the `traceparent` HTTP header:
```

File: src/platforms/common/performance/connect-services.mdx [hunk 2]
```diff
@@ -10,9 +10,13 @@ When tracing is enabled, the SDK will attach an outgoing header called `sentry-t
 
 SDKs with performance monitoring support listen to incoming requests and typically automatically pick up the incoming `sentry-trace` header to continue the trace (with the same `trace-id`) from there, connecting backend and frontend transactions into a single coherent trace using the `trace_id` value. Depending on the circumstance, this ID is transmitted either in a request header or in an HTML `<meta>` tag. All your transactions that have the same `trace-id` are connected. Linking transactions in this way makes it possible to navigate among them in [sentry.io](https://sentry.io) to better understand how the different parts of your system are affecting one another. You can learn more about this model in our [Distributed Tracing](/product/sentry-basics/tracing/distributed-tracing/) docs.
```
Comment (imatwawana) [inline src/platforms/common/performance/connect-services.mdx:11]: In the preview, this paragraph is really long. Does it make logical sense to break it up as I have below? If not, can you suggest a good break in that section?

```suggestion
SDKs with performance monitoring support listen to incoming requests and typically automatically pick up the incoming `sentry-trace` header to continue the trace (with the same `trace-id`) from there, connecting backend and frontend transactions into a single coherent trace using the `trace_id` value. Depending on the circumstance, this ID is transmitted either in a request header or in an HTML `<meta>` tag. 

All your transactions that have the same `trace-id` are connected. Linking transactions in this way makes it possible to navigate among them in [sentry.io](https://sentry.io) to better understand how the different parts of your system are affecting one another. You can learn more about this model in our [Distributed Tracing](/product/sentry-basics/tracing/distributed-tracing/) docs.
```

File: src/platforms/common/performance/connect-services.mdx [hunk 3]
```diff
@@ -10,9 +10,13 @@ When tracing is enabled, the SDK will attach an outgoing header called `sentry-t
...
 
+<PlatformSection notSupported={["javascript", "node"]}>
+
 If the instrumentation you are using doesn't automatically pick up the `sentry-trace` header, you can also continue a trace manually by using the `continueFromHeaders` function on a `Transaction`, which you can learn more about in our content for [the Transaction Interface](https://develop.sentry.dev/sdk/performance/#new-span-and-transaction-classes).
```
Comment (imatwawana) [inline src/platforms/common/performance/connect-services.mdx:15]:
```suggestion
If the instrumentation you're using doesn't automatically pick up the `sentry-trace` header, you can also continue a trace manually by using the `continueFromHeaders` function on a `Transaction`. Learn more about this in our [Transaction Interface](https://develop.sentry.dev/sdk/performance/#new-span-and-transaction-classes) documentation.
```

**Review by imatwawana — APPROVED** [review]
Made some tweaks to the original content and one small edit in what you've added.

File: src/platforms/common/performance/connect-services.mdx (hunk 2 above)
Comment (onurtemizkan) [reply src/platforms/common/performance/connect-services.mdx:11]: Yes, breaking like this makes sense to me.

**Review by AbhiPrasad — APPROVED** [review]

**Pull request merged**